pep-0001.txt

```diff
@@ -457,19 +457,30 @@ if they are never meant to be completed.  E.g. :pep:`1` (this PEP).
 PEP Maintenance
 ---------------
 
-In general, Standards track PEPs are no longer modified after they have
-reached the Final state. Once a PEP has been completed, the Language and
-Standard Library References become the formal documentation of the expected
-behavior.
+In general, PEPs are no longer substantially modified after they have reached
+the Final, Rejected or Superseded state. Once a PEP has been completed,
```

Comment:
This is also true (with only minor exceptions) for the Accepted state.

Reply:
Yeah; that was my impression and what the original wording of my Contributing Guide addition said, but when I moved it to PEP 1, the present wording there only described this for the "Final" state, and mentioned changes to "Accepted" PEPs on equal footing with "Provisional" ones. I've revised this and the passage below accordingly to reflect the current reality here.